packages/service-clients/end-to-end-tests/azure-client/src/test/.mocharc.cjs

```diff
@@ -8,4 +8,24 @@
 const packageDir = `${__dirname}/../..`;
 const getFluidTestMochaConfig = require("@fluid-internal/mocha-test-setup/mocharc-common");
 const config = getFluidTestMochaConfig(packageDir);
-module.exports = config;
+
+const args = process.argv.slice(2);
+
+function getFluidTestVariant() {
+	const driverIndex = args.indexOf("--driver");
+	const endpointIndex = args.indexOf("--driverEndpoint");
+
+	const testDriver = driverIndex !== -1 ? args[driverIndex + 1] : "";
+	const endpointName = endpointIndex !== -1 ? args[endpointIndex + 1] : "";
+
+	return `${testDriver}-${endpointName}`;
+}
+
+function getFluidTestMocha(packageDir, additionalRequiredModules = []) {
+	const testVariant = getFluidTestVariant();
+	process.env.FLUID_TEST_VARIANT = testVariant;
+
+	return config(packageDir, additionalRequiredModules, testVariant);
```

Comment:
this looks wrong, seems like you want `getFluidTestMochaConfig(packageDir, additionalRequiredModules, testVariant)` here. Same in other mocharc

Reply:
thanks for catching that!